frontend/pages/SoftwarePage/SoftwareVulnerabilityDetailsPage/SoftwareVulnOSVersions/_styles.scss

```diff
@@ -0,0 +1,5 @@
+.software-vuln-os-versions {
+  display: flex;
+  flex-direction: column;
```

Comment:
these are already applied by the parent `.card`, so you can just leave the `gap` here